dgraphtest/cluster.go

```diff
@@ -253,14 +253,14 @@ func (hc *HTTPClient) RunGraphqlQuery(params GraphQLParams, admin bool) ([]byte,
 
 // Backup creates a backup of dgraph at a given path
 func (hc *HTTPClient) Backup(c Cluster, forceFull bool, backupPath string) error {
-	// backup API was made async in the commit after this commit d3bf7b7b2786bcb99f02e1641f3b656d0a98f7f4
-	syncAPI, err := IsParent("caed2cb8c68b0ea8c69a3d71410a5fa6b0b94985", c.GetVersion())
+	// backup API was made async in the commit d3bf7b7b2786bcb99f02e1641f3b656d0a98f7f4
+	asyncAPI, err := IsParent("d3bf7b7b2786bcb99f02e1641f3b656d0a98f7f4", c.GetVersion())
 	if err != nil {
 		return errors.Wrapf(err, "error checking incremental restore support")
 	}
 
 	var taskPart string
-	if !syncAPI {
+	if asyncAPI {
```

Comment:
did we change the output of IsParent? Seems like we flipped the results?

Reply:
Parent terminology in Git world is actually confusing. Parents are usually the older commits whereas children are usually the newer commit. In our head, we usually think in terms of versions. I am thinking that I should rename this whole thing to compare versions instead of the Git child parent ancestor descendant thing. Let me work on a separate PR to improve this interface.